### PR TITLE
Updated the evaluation results after NF1 model retraining

### PR DESCRIPTION
**Pull request by MattsonCam**
We evaluate the NF1 model after retraining the model without plate 4 single cell. Feel free to add any constructive suggestions when reviewing this pr.

### CODE REVIEW

File: 2.evaluate_models/classify_genotypes/genotype_classification_evaluation.ipynb [hunk 1]
```diff
@@ -104,18 +104,18 @@
...
+     "shell.execute_reply": "2024-07-22T18:13:23.506434Z"
     },
     "jukit_cell_id": "zFHiLC64Ss"
    },
    "outputs": [],
    "source": [
     "data_path = pathlib.Path(f\"{root_dir}/1.train_models/classify_genotypes/data\")\n",
     "\n",
-    "evaldf = pd.read_parquet(f\"{data_path}/nf1_final_model_eval_data.parquet\")\n",
+    "evaldf = pd.read_parquet(f\"{data_path}/nf1_model_pre_evaluation_results.parquet\")\n",
```
Comment (jenna-tomkinson) [inline 2.evaluate_models/classify_genotypes/genotype_classification_evaluation.ipynb:118]: Just curious, why is it called "pre" evaluation results and not just evaluation?

**Review by jenna-tomkinson — APPROVED** [review]
LGTM! I don't see any big changes that would impact the eval results. I did leave one comment about naming for you to address prior to merging. Nice job!

File: 2.evaluate_models/classify_genotypes/genotype_classification_evaluation.ipynb (hunk 1 above)
Comment (MattsonCam) [reply 2.evaluate_models/classify_genotypes/genotype_classification_evaluation.ipynb:118]: This is to distinguish between the evaluation results, and the data to be used for creating those results

**Comment by MattsonCam** [comment]
Thanks @jenna-tomkinson, merging now!

**Pull request merged**